routes/v1/users.js

```diff
@@ -86,21 +86,21 @@ module.exports = function(/*middleware*/) {
 				return false;
 			}
 
-			var timestamp = parseInt(req.body.timestamp, 10) || undefined;
+			var timestamp = parseInt(req.body.timestamp, 10) || Date.now();
```

Comment:
I'm imagining `undefined` is fine as-is because if not set, `addMessage` should use `Date.now()`, but explicitly defining it here works too :smile: